app/controllers/HomeController.scala

```diff
@@ -140,7 +140,7 @@ class HomeController @Inject()(val homeView: views.html.Home,
       handleShowRequest(
         itvcErrorHandler = itvcErrorHandler,
         isAgent = false,
-        user.incomeSources.getCurrentTaxEndYear,
+        user.incomeSources.currentTaxYearEnd(dateService),
```

Comment:
it seems that we just calling another method/property from user.incomeSources~

Reply:
Done